<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="camera_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>监控显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 8pt &quot;Adobe Hebrew&quot;;
color: rgb(0, 255, 255);
border-color: rgb(85, 170, 255);
font: 75 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>实时监测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>420</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 8pt &quot;Adobe Hebrew&quot;;
color: rgb(0, 255, 255);
border-color: rgb(85, 170, 255);
font: 75 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>视频监测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 255, 255);
color: rgb(85, 255, 127);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>请输入文件绝对路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FILE_dir">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>330</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
